flows/prefect2_only/deployment_apply.py

```diff
@@ -0,0 +1,38 @@
+import anyio
+from packaging.version import Version
+
+import prefect
+from prefect.utilities.callables import parameter_schema
+
+
+@prefect.flow
+def hello(name: str = "world"):
+    prefect.get_run_logger().info(f"Hello {name}!")
+
+
+async def apply_deployment(deployment):
+    async with prefect.get_client() as client:
+        flow_id = await client.create_flow_from_name(deployment.flow_name)
+        await client.create_deployment(flow_id=flow_id, name=deployment.name)
+
+
+if __name__ == "__main__":
+    if Version(prefect.__version__) <= Version("2.19.0"):
+        from prefect.deployments import Deployment
```

Comment:
This moves `Deployment` import behind a version check as we won't have `Deployment` after 2.19.0. No other changes